cps/kni.c

```diff
@@ -1738,13 +1740,117 @@ loaded_kmod_matches_file(void *file, unsigned long len)
 	return false;
 }
 
+static int
+find_kni_kmod_path(char *path, size_t path_len, const char *alias)
+{
+	struct kmod_ctx *ctx;
+	char dirname[PATH_MAX];
+	struct utsname u;
+	struct kmod_list *l, *filtered, *list = NULL;
+	int ret;
+
+	/* Get kernel name and build module path. */
+	if (uname(&u) < 0) {
+		int saved_errno = errno;
+		CPS_LOG(ERR, "uname: %s\n", strerror(errno));
+		return -saved_errno;
+	}
+	ret = snprintf(dirname, sizeof(dirname),
+		"/lib/modules/%s", u.release);
+	RTE_VERIFY(ret > 0 && ret < (int)sizeof(dirname));
```

Comment:
Return an error here because `u.release` is outside of our control. `RTE_VERIFY()` is better suited when we don't ever expect to trigger it unless it's a bug.